nimx/naketools.nim

```diff
@@ -396,6 +396,8 @@ proc checkSdlRoot(b: Builder) =
 proc buildSDLForDesktop(b: Builder): string =
     when defined(linux):
         result = "/usr/lib"
+    when defined(haiku):
```

Comment:
This change will lead to assertion on linux now. It should be `elif` and probably in different order, if `linux` is defined along with `haiku`.

Reply:
good points. i have updated the  order as well as the logic to use `elif` instead of `when`. thank you for catching that!